Fixed: package.json scripts & fixed datatypes in tables & migrations

panchoarc opened the pull request:
###Summary
##Updated

- Fix package.json scripts for executing all seeds created.
- Change String Datatypes for TEXT Datatypes in order to supply better handling of larger texts (i.e: Content)

panchoarc committed:
Added: model & migration of members

panchoarc committed:
Added: Get all contacts endpoint

panchoarc committed:
Fixed: package.json seed scripts & fixed datatypes in tables

panchoarc committed:
Merge branch 'develop' into fix-scripts

rafalynch commented on src/models/activities.js:
```diff
@@ -1,7 +1,5 @@
+"use strict";
+const { Model } = require("sequelize");
 module.exports = (sequelize, DataTypes) => {
   class activities extends Model {
```
Podriamos aprovechar y cambiarle de activities a Activities

rafalynch replied:
Es un detalle, pero creo que las clases es mejor que empiecen con mayuscula

panchoarc replied:
Coincido, para poder distinguir si es un modelo o alguna otro elemento (route, controller,etc)

panchoarc replied:
Hecho los cambios sugeridos.

panchoarc committed:
Update activities.js 

Fixing typo in Activities for distinguish between a model or something else like route, controller, etc.

panchoarc committed:
Update activities.js
